packages/core/src/input/input.js

```diff
@@ -22,6 +24,11 @@ const Input = props => {
         value={searchTerm}
         onChange={onInputChange}
       />
+      {searchTerm !== "" && (
+        <span css={getStyles("clearInput", props)} onClick={onInputChange}>
```

Comment:
I think this should be a button element, not a span.
It's intended for a click.